chrome/content/zotero/xpcom/schema.js

```diff
@@ -2358,6 +2358,13 @@ Zotero.Schema = new function(){
 					yield Zotero.DB.queryAsync("INSERT OR IGNORE INTO itemRelations VALUES (?, ?, ?)", [newSubjectID, predicateID, newObjectURI]);
 				}
 			}
+			
+			else if (i == 90) {
+				yield Zotero.DB.queryAsync("ALTER TABLE feeds RENAME TO feedsOld");
+				yield Zotero.DB.queryAsync("CREATE TABLE feeds (\n    libraryID INTEGER PRIMARY KEY,\n    name TEXT NOT NULL,\n    url TEXT NOT NULL UNIQUE,\n    lastUpdate TIMESTAMP,\n    lastCheck TIMESTAMP,\n    lastCheckError TEXT,\n    cleanupAfter INT,\n    cleanupReadAfter INT,\n    refreshInterval INT,\n    FOREIGN KEY (libraryID) REFERENCES libraries(libraryID) ON DELETE CASCADE\n)");
+				yield Zotero.DB.queryAsync("INSERT INTO feeds SELECT libraryID, name, url, lastUpdate, lastCheck, lastCheckError, 30, cleanupAfter, refreshInterval FROM feedsOld");
+				yield Zotero.DB.queryAsync("DROP TABLE feedsOld");
+			}
```

Comment:
It doesn't matter too much for the beta, but technically either this should bump compatibility with `_updateCompatibility()` or the other code should test for presence of the columns (using `Zotero.DB.getColumns()`). In this case, if the other queries would just result in behind-the-scenes failures and broken purging, without crashing visible functionality, that's OK.

(DB compatibility will also matter substantially less after Z4Fx is gone.)